gateway/src/main/java/io/zeebe/gateway/impl/broker/BrokerClientImpl.java

```diff
@@ -97,8 +103,35 @@ public BrokerClientImpl(final GatewayCfg configuration, final ActorClock actorCl
     transport = transportBuilder.build();
     internalTransport = internalTransportBuilder.build();
 
+    final String[] parts = configuration.getCluster().getContactPoint().split(":");
+    final Integer port = Integer.parseInt(parts[1]);
+
+    atomix =
+        AtomixCluster.builder()
+            .withMemberId("gateway")
+            .withClusterId(configuration.getCluster().getClusterName())
+            .withMembershipProvider(
+                BootstrapDiscoveryProvider.builder()
+                    .withNodes(Address.from(parts[0], (port + CLIENT_ADDR_OFFSET)))
```

Comment:
This has to be configurable and not a magic calculation

Reply:
This should be the contact point from the gateway cluster configuration https://github.com/zeebe-io/zeebe/blob/ad64abcd724cf610e7bcefb14218170f06abcd33/gateway/src/main/java/io/zeebe/gateway/impl/configuration/ClusterCfg.java#L47.

We then should remove the registration of the contact point to the endpoint manager, this will happen automagically when we receive the memberships events over atomix, i.e. delete this:
https://github.com/zeebe-io/zeebe/blob/ad64abcd724cf610e7bcefb14218170f06abcd33/gateway/src/main/java/io/zeebe/gateway/impl/broker/BrokerClientImpl.java#L112-L114